govc/events/command.go

```diff
@@ -45,6 +48,8 @@ func (cmd *events) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DatacenterFlag.Register(ctx, f)
 
 	f.IntVar(&cmd.Max, "n", 25, "Output the last N events")
+	f.BoolVar(&cmd.Tail, "t", false, "Tail events")
```

Comment:
ok, let's go with '-f' then.  I don't expect we'd add a file output option for this command, if we did then it would be '-o' (go build, curl and strace come to mind as examples using -o).